web_app_drawer/static/src/less/variables.less

```diff
@@ -0,0 +1,15 @@
+/* Copyright 2016 LasLabs Inc.
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
+
+// App Drawer / Icons
+
+@app-drawer-icon-size:      5.625em;
+@app-drawer-icon-margin:    0.625em;
+@app-drawer-width:          80%;
+@app-drawer-title-height:   2.9em;
+
+// Navbar
+
+@navbar-height:  2.3em;
+@navbar-padding-vertical: .565em;
+@navbar-padding-horizontal: .565em;
```

Comment:
Good call with the math on the height - I forget we're in LESS world now. Forgive while I shed oh so many years of CSS 😛 

Awesome, I'll take all the help I can get. I've been toying with tossing you on the contributors list for all the reviews and ideas anyways, now you can do it yourself!

Quitar raya- I used a panel for the Apps, which is creating the line. My reasoning behind this is that we can then make a `web_drawer_widget`, which will provide a widget panel to add in whatever we feel like.

Could be overboard or too much forward thinking, but it seems to me widgets would be a great use of wasted space there 😄